bundle/manifests/kuadrant-operator.clusterserviceversion.yaml

```diff
@@ -262,6 +262,14 @@ spec:
           - patch
           - update
           - watch
+        - apiGroups:
```

Comment:
why do we need this permission now?

Reply:
It's required for https://github.com/Kuadrant/kuadrant-operator/pull/550/files#diff-c1042ac0cb48fd02920d56c5a6426812656d90081386ed7dbefda245a0e0410cR157